packages/devextreme/testing/tests/DevExpress.ui.events/events.utils.tests.js

```diff
@@ -505,6 +506,55 @@ QUnit.module('event utils', () => {
     });
 });
 
+QUnit.module('getEventTarget module', () => {
+    test('getEventTarget returns the element on which the event was raised', function(assert) {
+        assert.expect(1);
+
+        const host = document.getElementById('qunit-fixture');
+        const div = document.createElement('div');
+
+        host.appendChild(div);
+
+        const customEvent = new Event('customEvent', {
+            bubbles: true,
+            composed: true,
+        });
+
+        host.addEventListener('customEvent', (event) => {
+            const target = getEventTarget(event);
+
+            assert.strictEqual(div, target, 'getEventTarget returned target correctly');
+        });
+
+        div.dispatchEvent(customEvent);
+    });
+
+    if(QUnit.urlParams['nojquery']) {
+        skipInShadowDomMode('getEventTarget returns the element on which the event was raised if there is ShadowDOM', function(assert) {
+            assert.expect(1);
+
+            const shadowHost = document.getElementById('qunit-fixture');
+            const shadowRoot = shadowHost.attachShadow({ mode: 'open' });
```

Comment:
[test isolation]
oh, it's better never do smth with `qunit-fixture` because it does not change between test run, only its content is updated. Not i believe qunit-fixture is always shadowed after this tests is run